code/modules/spells/aoe_turf/conjure/forcewall.dm

```diff
@@ -51,3 +51,17 @@
 	new /obj/effect/forcefield/cult(get_turf(src))
 	qdel(src)
 	return
+
+/obj/effect/forcefield/invisible_wall
+	name = "invisible wall"
+	desc = "The nemesis of explorers since the dawn of times."
+
+/obj/effect/forcefield/invisible_wall/New()
+	..()
+
+	alpha = 0
```

Comment:
Prevents it from being invisible in the map editor due to alpha = 0

You want to actually see the invisible walls when you're working with them